File: composeApp/src/androidMain/kotlin/org/gce/racehub/forgotpassword/ForgotPasswordState.kt
package org.gce.racehub.forgotpassword

enum class ForgotPasswordStep { REQUEST, CONFIRM }

data class ForgotPasswordState(
    val step: ForgotPasswordStep = ForgotPasswordStep.REQUEST,
    val email: String = "",
    val otp: String = "",
    val newPassword: String = "",
    val confirmPassword: String = "",
    val isPasswordVisible: Boolean = false,
    val isLoading: Boolean = false,
    val errorMessage: String? = null
)

File: iosApp/iosApp/ForgotPassword/ForgotPasswordState.swift
import Foundation

enum ForgotPasswordStep {
    case request
    case confirm
}

struct ForgotPasswordState {
    var step: ForgotPasswordStep = .request
    var email: String = ""
    var otp: String = ""
    var newPassword: String = ""
    var confirmPassword: String = ""
    var isPasswordVisible: Bool = false
    var isLoading: Bool = false
    var errorMessage: String? = nil
}
